models/review.js

```diff
@@ -0,0 +1,29 @@
+const mongoose = require('mongoose');
+const reviewSchema = new mongoose.Schema({
+    recipe: {
+        type: mongoose.Schema.Types.ObjectId,
+        ref: 'Recipe',
+        required: true
+    },
+    user: {
+        type: mongoose.Schema.Types.ObjectId,
+        ref: 'User',
+        required: true
+    },
```

Comment:
```suggestion
    recipeId: {
        type: mongoose.Schema.Types.ObjectId,
        ref: 'Recipe',
        required: true
    },
    userId: {
        type: mongoose.Schema.Types.ObjectId,
        ref: 'User',
        required: true
    },
```

Any referenced field should have Id included in the name.
We will know at a glance.